src/main.lisp

```diff
@@ -215,45 +221,51 @@
 )
 
 ; testing
-(gamekit:bind-button :right :repeating
-  (lambda () (incf (gamekit:x (rect *block-a*)) 2)))
-
-(gamekit:bind-button :left :repeating
-  (lambda () (decf (gamekit:x (rect *block-a*)) 2)))
-
-(gamekit:bind-button :up :repeating
-  (lambda () (incf (gamekit:y (rect *block-a*)) 2)))
-
-(gamekit:bind-button :down :repeating
-  (lambda () (decf (gamekit:y (rect *block-a*)) 2)))
-
-; Input bindings
-(gamekit:bind-button :a :pressed
-  (lambda () (setf *move-dir* -1)))
-
-(gamekit:bind-button :a :released
-  (lambda () (setf *move-dir* 0)))
-
-(gamekit:bind-button :d :pressed
-  (lambda () (setf *move-dir* 1)))
-
-(gamekit:bind-button :d :released
-  (lambda () (setf *move-dir* 0)))
-
-(gamekit:bind-button :o :released
-  (lambda () (setf *debug* (not *debug*))))
-
-(gamekit:bind-button :space :pressed
-  (lambda ()
-    (case *game-state*
-      (0
-        (setf *transitioning* t)
-      )
-
-      (1
-          (when *grounded*
+(defmethod post-initialize ((this moppu))
+  (gamekit:bind-button
```

Comment:
All input binding is better to be placed inside `#'post-initialize`

Reply:
This is what I have been loking for, trying to bind keys without start was not possible.